sfge/src/main/java/com/salesforce/rules/fls/apex/operations/ValidationConverter.java

```diff
@@ -213,10 +222,24 @@ private ApexValue<?> getObjectValue(ApexValue<?> apexValue) {
         return apexValue;
     }
 
+    /** check if validation is necessary. if so, add it to holder. */
     private void convertSoqlQueryInfo(
             ValidationHolder holder,
             ProcessFields defaultProcessFields,
             HashSet<SoqlQueryInfo> queryInfos) {
+        convertSoqlQueryInfo(holder, defaultProcessFields, queryInfos, false);
```

Comment:
TO CONFIRM: is there any other situation, other than in line 160, where the `isUserModeAccessLevel` boolean should be used? The boolean is used to override a SOQL statement's access level. So far, it is only used in methods like `Database.query(String queryString, AccessLevel lvl)` where `lvl` is a form of override.

Reply:
Sorry for the delay. I don't know of any other scenarios where that parameter would need to be used.
SOQL/DML vertices have their own handling for their user-mode syntax elsewhere.